templates/trivy/trivy-sts.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if .Values.trivy.enabled }
```

Comment:
I think you missed a curly bracket here ;)
```suggestion
{{- if .Values.trivy.enabled }}
```

Reply:
Oh my, sorry .. :) 👍